docroot/nbproject/project.xml

```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@dishabhadra Can you also add the folder nbproject to gitignore. So that it doesn't get pushed off by anyone else in future commits?